cluster/ci/monitoring/mixins/prometheus/build_cop_alerts.libsonnet

```diff
@@ -0,0 +1,26 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'build-cop-target-low',
+        rules: [
+          {
+            alert: '%s-low' % job_name_regex,
+            expr: |||
+              sum(rate(prowjob_state_transitions{job="plank",job_name=~"%s",job_name!~"rehearse.*",state="success"}[30m]))/sum(rate(prowjob_state_transitions{job="plank",job_name=~"%s",job_name!~"rehearse.*",state=~"success|failure"}[30m])) < 1
+            ||| % [job_name_regex, job_name_regex],
+            'for': '5m',
+            labels: {
+              severity: 'slack',
+              team: 'build-cop',
+            },
+            annotations: {
+              message: '%s jobs have been below the target (100%%) for 5 minutes.' % job_name_regex,
```

Comment:
Can we get the regex in a code block?

```
'`%s` jobs have been below the target
```

Reply:
Trying.

Reply:
https://coreos.slack.com/archives/CHY2E1BL4/p1561141209042700
Done